sw/device/tests/dif/dif_plic_unittest.cc

```diff
@@ -373,5 +373,90 @@ TEST_F(IrqCompleteTest, Target0Success) {
   }
 }
 
+class SoftwareIrqForceTest : public PlicTest {
+ protected:
+  // Make sure our tests are up to date.
+  SoftwareIrqForceTest() { EXPECT_EQ(RV_PLIC_PARAM_NUMTARGET, 1); }
+};
+
+TEST_F(SoftwareIrqForceTest, Target0NullArgs) {
+  EXPECT_EQ(dif_plic_software_irq_force(nullptr, kTarget0), kDifPlicBadArg);
+}
+
+TEST_F(SoftwareIrqForceTest, Target0BadTarget) {
+  EXPECT_EQ(dif_plic_software_irq_force(&dif_plic_, RV_PLIC_PARAM_NUMTARGET),
+            kDifPlicBadArg);
+}
```

Comment:
nit: I know these are consistent with the names of the other tests in this file but IMHO `Target0` is not relevant here. In the first test, second argument should just be a valid target and in the second one it is invalid anyway. Applies to other tests as well.

Reply:
Yes, that is a good point.

Reply:
Done.